examples/06-kitchen-sink/src/block/block.js

```diff
@@ -0,0 +1,482 @@
+	AlignmentToolbar,
+	BlockControls,
+	MediaUpload,
+	InnerBlocks,
+	PanelColorSettings,
+	InspectorControls,
+	URLInput
+} = wp.editor;
+
+// Import Inspector components
+const {
+	IconButton,
+	Toolbar,
+	Button,
+	PanelBody,
+	PanelRow,
+	ToggleControl,
+	RangeControl,
+	TextControl,
+	TextareaControl,
+	SelectControl
+} = wp.components;
+
+// Declare attributes to save various data in block: https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-attributes/
+const kitchenAttributes = {
+	exampleToggle: {
+		type: "boolean",
+		default: false
+	},
+	exampleRange: {
+		type: "number",
+		default: 18
+	},
+	exampleSelect: {
+		type: "string",
+		default: "option_1"
+	},
+	exampleText: {
+		type: "string"
+	},
+	exampleTextarea: {
+		type: "string"
+	},
+	exampleColor: {
+		type: "string",
+		default: "#FFFFFF"
+	},
+	exampleAlignment: {
+		type: "string"
+	},
+	exampleRichH2: {
+		type: "string",
+		source: "children",
+		selector: "h2"
+	},
+	exampleRichParagraph: {
+		type: "string",
+		source: "html",
+		selector: "p"
+	},
+	exampleRichList: {
+		type: "string",
+		selector: "ul"
+	},
+	examplePlainText: {
+		type: "string"
+	},
+	exampleLink: {
+		type: "string"
+	},
+	exampleImageAlt: {
+		type: "string"
+	},
+	exampleImageURL: {
+		type: "string",
+		source: "attribute",
+		attribute: "src",
+		selector: "img"
+	},
+	exampleWidth: {
+		type: "string",
+		default: ""
+	}
+};
+
+/**
+ * Register: aa Gutenberg Block.
+ *
+ * Registers a new block provided a unique name and an object defining its
+ * behavior. Once registered, the block is made editor as an option to any
+ * editor interface where blocks are implemented.
+ *
+ * @link https://wordpress.org/gutenberg/handbook/block-api/
+ * @param  {string}   name     Block name.
+ * @param  {Object}   settings Block settings.
+ * @return {?WPBlock}          The block, if it has been successfully
+ *                             registered; otherwise `undefined`.
+ */
+registerBlockType("cgb/block-kitchen-sink", {
+	// Block name. Block names must be string that contains a namespace prefix. Example: my-plugin/my-custom-block.
+	title: __("kitchen-sink - CGB Block"), // Block title.
+	icon: "shield", // Block icon from Dashicons → https://developer.wordpress.org/resource/dashicons/.
+	category: "create-guten-block-kitchen", // Custom Block category: Look in src/init.php on Line 67. You can also use default groups E.g. common, formatting, layout widgets, embed.
+	keywords: [
+		__("kitchen-sink — CGB Block"),
+		__("CGB Example"),
+		__("create-guten-block")
+	],
+	supports: {
+		// Adds block supports: https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-registration/#supports-optional
+		align: true,
+		alignWide: true,
+		anchor: false,
+		customClassName: true,
+		className: true,
+		html: true,
+		inserter: true,
+		multiple: true,
+		reusable: true
+	},
+	attributes: kitchenAttributes, // Use attributes from line 42
```

Comment:
I don't see a benefit in creating `kitchenAttributes` constant. Instead just add it all here.